### PR TITLE
Polymer 1.0 port

### PR DESCRIPTION
**Pull request by atotic**
Ported to Polymer 1.0

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -29,13 +29,13 @@
   }
   </style>
 </head>
-<body layout vertical center-justified center unresolved>
+<body layout vertical center-justified center>
```
Comment (ebidel) [inline demo/index.html]: `class="layout vertical center-justified center"` now

File: geo-location.html [hunk 2]
```diff
@@ -43,141 +40,149 @@
...
-         *
-         * @attribute watchPos
-         * @type boolean
-         * @default false
          */
-        watchPos: false,
+        watchPos: {
+          type: Boolean,
+          value: false,
+          observer: "_watchPosChanged"
```
Comment (ebidel) [inline geo-location.html]: single quotes

File: geo-location.html [hunk 3]
```diff
@@ -43,141 +40,149 @@
...
+          type: Number,
+          value: 5000
+        },
+
+        /**
+         * Geolocation API position object
+         */
+        position: {
+          type: Object,
+          notify: true
```
Comment (ebidel) [inline geo-location.html]: we should make this `readOnly: true`

File: geo-location.html [hunk 4]
```diff
@@ -43,141 +40,149 @@
...
-</polymer-element>
+      },
+
+      /**
+       * Success callback when the Geolocation API returns results.
+       *
+       * @param {Position} pos A position object from the Geolocation API.
+       */
+      _onPosition: function(pos) {
+        this.position = pos;
```
Comment (ebidel) [inline geo-location.html]: then you need `this._setPosition(pos);`

File: geo-location.html [hunk 5]
```diff
@@ -43,141 +40,149 @@
...
-           * @attribute longitude
-           * @type Number
-           * @default null
-           */
-          longitude: {value: null, reflect: true}
+      properties: {
+        /**
+         * The latitude of the current position.
+         */
+        latitude: {
```
Comment (ebidel) [inline geo-location.html]: should we make lat/lng `readOnly:true`?

File: geo-location.html [hunk 6]
```diff
@@ -43,141 +40,149 @@
...
+
+      /**
+       * Success callback when the Geolocation API returns results.
+       *
+       * @param {Position} pos A position object from the Geolocation API.
+       */
+      _onPosition: function(pos) {
+        this.position = pos;
+        this.latitude = pos.coords.latitude;
+        this.longitude = pos.coords.longitude;
```
Comment (ebidel) [inline geo-location.html]: will need the special setter here too

**Comment by ebidel** [comment]
Thank you for doing this!

File: geo-location.html (hunk 5 above)
Comment (atotic) [inline geo-location.html]: Should I kill reflectToAttribute for these? I used it for debugging, it is not that useful

**Comment by atotic** [comment]
No need to thank, I needed a port. Glad it was you :)

File: demo/index.html (hunk 1 above)
Comment (atotic) [inline demo/index.html]: I thought unresolved was gone in 1.0

Comment (ebidel) [inline demo/index.html]: it appears to be back, at least testing today in 1.0.3. But my comment was also to move the layout attributes to classes.

File: geo-location.html (hunk 5 above)
Comment (ebidel) [inline geo-location.html]: I recall people wanting the values to reflect in 0.5. Let's keep.

File: demo/index.html (hunk 1 above)
Comment (atotic) [inline demo/index.html]: Huh?

Comment (ebidel) [inline demo/index.html]: You want: `<body class="layout vertical center-justified center" unresolved>`

**Comment by atotic** [comment]
fixes in, PTAL

**Comment by ebidel** [comment]
LGTM!

**Pull request merged**